bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerUnderreplicationManager.java

```diff
@@ -29,6 +28,40 @@
  * Interface for marking ledgers which need to be rereplicated.
  */
 public interface LedgerUnderreplicationManager extends AutoCloseable {
+
+    /**
+     * UnderReplicated ledger representation info.
+     */
+    class UnderreplicatedLedger {
```

Comment:
move this out as a separate class file in this package

Reply:
done